packages/grid/_modules_/grid/components/panel/Panel.tsx

```diff
@@ -57,7 +57,16 @@ export function Panel(props: PanelProps) {
   }
 
   return (
-    <Popper placement="bottom-start" open={open} anchorEl={anchorEl}>
+    <Popper
+      placement="bottom-start"
+      open={open}
+      anchorEl={anchorEl}
+      modifiers={{
+        flip: {
+          enabled: false,
```

Comment:
👍

Reply:
Mind that this will need special handling between Material-UI v4 and v5

Reply:
Yup, I remember that you suggested to disable it. I'll check how to handle it for v5